src/app/components/user/cognito-user-view.jsx

```diff
@@ -0,0 +1,127 @@
+import React, { useEffect, useState } from 'react';
+import {
+  func,
+} from 'prop-types';
+import CheckBoxIcon from '@material-ui/icons/CheckBox';
+import CheckBoxOutlineBlankOutlinedIcon from '@material-ui/icons/CheckBoxOutlineBlankOutlined';
+import {
+  Card,
+  CardContent,
+  CardHeader,
+  ThemeProvider,
+  Typography,
+  makeStyles,
+} from '@material-ui/core';
+
+import theme from '../../themes/theme';
+import {
+  user as userPropType,
+} from '../../shared/prop-types';
+
+const useStyles = makeStyles(() => ({
```

Comment:
The extra function here isn't necessary. It started out that way, but I've been trying to remove it everywhere I find it:

```suggestion
const useStyles = makeStyles({
```

Then end of the makeStyles will need a `)` removed, I believe